verilog/tools/ls/symbol-table-handler.cc

```diff
@@ -0,0 +1,161 @@
+#include "verilog/tools/ls/symbol-table-handler.h"
+
+#include <filesystem>
+
+#include "common/strings/line_column_map.h"
+
+namespace verilog {
+
+static const std::string fileschemeprefix = "file://";
+
+bool LSPUriToPath(absl::string_view uri, std::string &path) {
+  auto found = uri.find(fileschemeprefix);
+  if (found == std::string::npos) {
+    return false;
+  }
+  if (found != 0) {
+    return false;
+  }
+  std::string res{uri.substr(found + fileschemeprefix.size())};
+  path = res;
+  return true;
+}
+
+bool PathToLSPUri(absl::string_view path, std::string &uri) {
```

Comment:
similar comment with writable output parameters to use a pointer (`std::string *uri`).